bootstrap.ps1

```diff
@@ -0,0 +1,20 @@
+param(
+    [switch]$build = $false
+)
+
+# Make sure we're in the repository root.
+cd "$PSScriptRoot"
+
+git fetch --tags --force --quiet
+
+$version = $( git tag --sort=committerdate | Select-Object -Last 1 )
+
+if ($build) {
+    git checkout $version
+    .\scripts\install.bat
+} else {
+    curl -o tigerbeetle.zip "https://github.com/tigerbeetledb/tigerbeetle/releases/download/$version/tigerbeetle-x86_64-windows-$version.zip"
```

Comment:
In both cases, should we `rm` this file when done? Perhaps using `New-TemporaryFile` / `mktemp` too? Alternatively if we switch to `gzip` we can `| gunzip` directly (not sure about PowerShell though...)